src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -373,30 +388,44 @@ def load_pytorch_state_dict_in_tf2_model(
     if tf_model._keys_to_ignore_on_load_unexpected is not None:
         for pat in tf_model._keys_to_ignore_on_load_unexpected:
             unexpected_keys = [k for k in unexpected_keys if re.search(pat, k) is None]
+    if not skip_logger_warnings:
+        _log_key_warnings(missing_keys, unexpected_keys, mismatched_keys, class_name=tf_model.__class__.__name__)
+
+    if output_loading_info:
+        loading_info = {
+            "missing_keys": missing_keys,
+            "unexpected_keys": unexpected_keys,
+            "mismatched_keys": mismatched_keys,
+        }
+        return tf_model, loading_info
+
+    return tf_model
 
+
+def _log_key_warnings(missing_keys, unexpected_keys, mismatched_keys, class_name):
```

Comment:
nit - definition should go above the lines of code where it's used i.e. before load_pytorch_state_dict_in_tf2_model

Reply:
Done!